src/app/archive/lib/processor.ml

```diff
@@ -421,12 +517,69 @@ module Zkapp_states = struct
     ; element5 : int
     ; element6 : int
     ; element7 : int
+    ; element8 : int
```

Comment:
This might be doable, or not doable. But could we use dependently typed vector to model Zkapp_states here? It seems very repetitive